topics/contributing/tutorials/fair-by-design/tutorial.md

```diff
@@ -0,0 +1,1065 @@
+      <div class="card-header"><i class="fa fa-fast-backward" aria-hidden="true"></i> Think backward</div>
+      <div class="card-body">
+        <h5 class="card-title">Step 3</h5>
+        <p class="card-text">How should you structure the material to provide effective learning experience?</p>
+      </div>
+    </div>
+  </div>
+
+</div>
+><tip-title>Learn more about the backward learning process</tip-title>
+> - [Steps of the backward learning process](https://fair-by-design-methodology.github.io/FAIR-by-Design_ToT/latest/Stage%201%20%E2%80%93%20Prepare/02-Preparing%20FAIR%20Learning%20Objects/02-Preparing%20FAIR%20Learning%20Objects_cont/#backward-instructional-design-process)
+{: .tip }
+
+### Define
+
+> <question-title>Purpose</question-title>
+> When and how the learning materials can be used and for what purposes?
+{: .question }
+
+> <question-title>Target Audience</question-title>
+> Is there anything specific that needs to be taken into account, such as cultural context?
+{: .question }
+
+> <question-title>Prerequisites</question-title>
+> What does the target audience need to know or understand before starting the learning process?
+{: .question }
+
+> <question-title>Scope</question-title>
+> Is it going to be a single learning unit, or a group such as a course?
+{: .question }
+
+> <question-title>Learning Objectives</question-title>
+> What competences will be gained after successful completing of the learning process?
+>
+> ><tip-title>Be SMART</tip-title>
+> >Objectives should be **s**pecific, **m**easurable, **a**ttainable, **r**elevant and **t**ime-bound.
+> {: .tip}
+>
+> ><tip-title>Use Blooms Taxonomy</tip-title>
+> >Formulate the objectives as actionable verb + observable knowledge, skill, attitude, behavior or ability.
+> {: .tip}
+> 
+> [Read more](https://fair-by-design-methodology.github.io/FAIR-by-Design_ToT/latest/Stage%201%20%E2%80%93%20Prepare/02-Preparing%20FAIR%20Learning%20Objects/02-Preparing%20FAIR%20Learning%20Objects_cont/#defining-learning-objectives)
+{: .question }
+
+> <details-title>More details on ideating learning outcomes and related information</details-title>
+> [Read about defining intended Learning Outcomes in the Design and plan session, course, materials tutorial available on GTN]( {% link topics/contributing/tutorials/design/tutorial.md#define-intended-learning-outcomes-los %} )
+{: .details}
+
+<a href="https://fair-by-design-methodology.github.io/FAIR-by-Design_Book/4%20-%20FAIR-by-design%20learning%20materials%20creation/4.1%20-%20Workflow%20stages%20description/411-prepare/" class="btn btn-dark text-white btn-lg btn-block">Go to the full description of FAIR-by-Design Methodology: Prepare stage ...</a>
+
+## Stage 2 - Discover
+
+> Greater even than the greatest discovery is to keep open the way to future discovery.
+{: .quote author="John Jacob Abel"}
+### Get inspired
+
+Reusable materials can be found anywhere. These are just some examples:
+
+> <code-in-title>GTN</code-in-title>
```

Comment:
the 'code-in' boxes are meant for coding exercises, was there a particular reason you chose these? A comment or tip box might make more sense

Reply:
We have removed the 'code-in' boxes with lists